docker-compose.yml

```diff
@@ -170,9 +170,13 @@ services:
       ORB_DRIVER_HOST_URL: ${ORB_DRIVER_HOST_URL}
       ORB_DRIVER_TLS_SYSTEMCERTPOOL: ${ORB_DRIVER_TLS_SYSTEMCERTPOOL}
     ports:
-      - "8121:8121"
+      - "8122:8121"
```

Comment:
Pls revert this to `8121:8121`

Reply:
it's a bug you have two docker running on the same port

```  driver-didkit:
    image: ghcr.io/spruceid/didkit-http:latest
    environment:
      PORT: 8080
      HOST: 0.0.0.0
    ports:
      - "8121:8080"
  eosio-driver:
    container_name: eosio-driver
    image: gimlyblockchain/eosio-universal-resolver-driver
    ports:
      - "8123:8080"
  orb-did-driver:
    container_name: orb-did-driver
    image: ghcr.io/trustbloc/orb-did-driver:v0.1.3
    environment:
      ORB_DRIVER_HOST_URL: ${ORB_DRIVER_HOST_URL}
      ORB_DRIVER_TLS_SYSTEMCERTPOOL: ${ORB_DRIVER_TLS_SYSTEMCERTPOOL}
    ports:
      - "8122:8121"
    command: start``` 

the reason I changed to 8122 is that there were two packages with the 8121.

Reply:
Think you had an outdated version but thx for pointing out.